assets/src/dashboard/app/views/myStories/pageHeading.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { ViewHeader } from '../../../components';
+import MyStoriesSearch from './myStoriesSearch';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 40px 20px;
+`;
+
+const ViewHeaderContainer = styled.div`
+  width: 60%;
+  margin: auto 0;
+`;
+
+const SearchContainer = styled.div`
+  position: absolute;
+  max-width: 35%;
+  margin: auto 0;
+  right: 20px;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const PageHeading = ({
+  defaultTitle,
+  filteredStories = [],
+  handleTypeaheadChange,
+  typeaheadValue = '',
+  isNoResultsText,
```

Comment:
Hmmm... I'm not sure you need `isNoResultsText`.  

`PageHeading` is already being passed in the `filteredStories` and the current `typeaheadValue`, couldn't you use `filteredStories.length > 0` to figure out if `resultsText` should be "Results" or "No results" and then the rest of the logic should work as expected?

Reply:
yes i suppose so. I opted for isNoResultsText because I'm doing the same check below to show the grid so i thought keeping the logic in 1 spot would be beneficial. But it does add 1 more prop to get passed through. I can update it.